.github/workflows/_ci.yaml

```
@@ -476,15 +476,70 @@ jobs:
   #     ARTIFACTS: |
   #       test-equinox.log
   #   secrets: inherit
+  test-te-h100:
+    needs: build-jax
+    if: inputs.ARCHITECTURE == 'amd64'
+    uses: ./.github/workflows/_transformer_engine_eks.yaml
+    with:
+      JAX_IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }}
+      JOB_NAME: transformerengine-${{ github.run_id }}
+      S3_BUCKET: jax-toolbox-eks-output
+      CI_NAME: transformer-engine
+    secrets: inherit
 
-  # test-te-multigpu:
-  #   needs: build-upstream-pax
-  #   if: inputs.ARCHITECTURE == 'amd64' # arm64 runners n/a
-  #   uses: ./.github/workflows/_test_te.yaml
-  #   with:
-  #     TE_IMAGE: ${{ needs.build-upstream-pax.outputs.DOCKER_TAG_FINAL }}
-  #   secrets: inherit
+  test-te-unit-a100:
+    needs: build-jax
+    secrets: inherit
+    if: inputs.ARCHITECTURE == 'amd64'  # arm64 runners n/a
+    uses: ./.github/workflows/_test_unit.yaml
+    with:
+      TEST_NAME: te
+      EXECUTE: |
+        docker run -i --gpus all --shm-size=1g -v $PWD:/log \
+        ${{ needs.build-jax.outputs.DOCKER_TAG_FINAL }} \
+        bash <<"EOF" |& tee test-te.log
+          set -x
 
+          pip install pytest-reportlog pytest-xdist
+          # Start MPS daemon
+          nvidia-cuda-mps-control -d
+          # TE's default is slightly different, without the hyphen
+          export TE_PATH=${SRC_PATH_TRANSFORMER_ENGINE}
+          # 1 GPU per worker, 6 workers per GPU
+          pytest-xdist.sh 1 6 /log/pytest-report-L0-unittest.jsonl bash ${TE_PATH}/qa/L0_jax_unittest/test.sh
```

```suggestion
          pytest-xdist.sh 1 3 /log/pytest-report-L0-unittest.jsonl bash ${TE_PATH}/qa/L0_jax_unittest/test.sh
```
I see OOM errors in the logs, I think 6 was based on 80GB cards and the A100s are 40GB